!GameTexts/NewTexts.xml

```diff
@@ -201,16 +191,16 @@
 			<Text>Siege Mobility</Text>
 		</Row>
 		<Row Tag="TXT_KEY_PROMOTION_UNIT_ROME_LEGATUS_LEGIONIS_EFFECT_HELP">
-			<Text>Double [ICON_ARROW_RIGHT] Movement [COLOR_POSITIVE_TEXT]on Flat, but not hindering Terrain from Great General[ENDCOLOR].</Text>
+			<Text>Double [ICON_ARROW_RIGHT] Movement [COLOR_POSITIVE_TEXT]on Open Terrain if within 2 tiles of a Great General[ENDCOLOR].</Text>
 		</Row>
 		
-		<!-- Dhanuraashi -->
+		<!-- Dhanurgraha -->
 		<!-- Created by adan_eslavo on 4/12/2017 -->
 		<Row Tag="TXT_KEY_PROMOTION_UNIT_INDIA_EPIC">
 			<Text>Epic</Text>
 		</Row>
 		<Row Tag="TXT_KEY_PROMOTION_UNIT_INDIA_EPIC_HELP">
-			<Text>Unit gains [ICON_CULTURE] Culture and additional XP [COLOR_POSITIVE_TEXT]on Kill[ENDCOLOR]. Yields doubled [COLOR_POSITIVE_TEXT]on Barbarian Kills[ENDCOLOR].</Text>
+			<Text>Unit gains [ICON_CULTURE] Culture and additional XP [COLOR_POSITIVE_TEXT]on Kill[ENDCOLOR]. Culture doubled [COLOR_POSITIVE_TEXT]on Barbarian Kills[ENDCOLOR].</Text>
```

Comment:
"Yields" was used to not repeat word Culture. But if you want to add it then add also [ICON_CULTURE] before.

Reply:
I disagree, because people might interpret XP as a yield, and I don't think that XP is doubled to 10 vs barbarians, right?

you could add [ICON_CULTURE], but it might be unnecessary since it is so close to the previous mention of it. Your call